django/http/multipartparser.py

```diff
@@ -72,13 +72,17 @@ def __init__(self, META, input_data, upload_handlers, encoding=None):
 
         content_type = META.get('CONTENT_TYPE', '')
         if not content_type.startswith('multipart/'):
-            raise MultiPartParserError('Invalid Content-Type: %s' % content_type)
+            raise MultiPartParserError(
```

Comment:
Some of these changes are okay but I would change places like this. As noted in the [style guide](https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/#python-style), we allow lines up to 119 characters when it helps readability.